themes/bootstrap3/templates/RecordDriver/DefaultRecord/result-list.phtml

```diff
@@ -172,7 +176,7 @@
         </span>
       <?php endif; ?>
 
-      <?php if ($this->cart()->isActiveInSearch() && $this->params->getOptions()->supportsCart() && $this->cart()->isActive()): ?>
+      <?php if ($this->cart()->isActiveInSearch() && isset($params) && $this->params->getOptions()->supportsCart() && $this->cart()->isActive()): ?>
```

Comment:
We use `$params` and `$this->params` in the same line; should we pick one for consistency?

Reply:
Fixed.